src/pkg/zoci/push.go

```diff
@@ -110,5 +111,8 @@ func annotationsFromMetadata(metadata *v1alpha1.ZarfMetadata) map[string]string
 		annotations[ocispec.AnnotationVendor] = vendor
 	}
 
+	// annotations explicitly defined in `metadata.annotations` take precedence over legacy fields
+	maps.Copy(annotations, metadata.Annotations)
+
```

Comment:
I could not find any existing tests for populating annotations on the OCI manifest and config. Happy to add some if there is an obvious place to do that.

Reply:
Yeah there aren't currently any tests that verify that annotations end up on the manifest or manifest config